packages/card/README.md

```diff
@@ -0,0 +1,133 @@
+# React Card
+
+MDC React Card is a component for material design cards.
+
+## Installation
+
+```
+npm install @material/react-card
```

Comment:
Are we publishing this to NPM right now? Or are you just planning ahead?

Also, the package names seem inconsistent; there's `react-card` and `react-fab`, but also `material-icon-react` and `top-app-bar-react`. Which one are we going with? (I vote for `react-foo`)

Reply:
Just planning ahead.

And yes this PR is inconsistent, and is fixed in #16 for `react-foo` :)